src/current/v23.2/physical-cluster-replication-technical-overview.md

```diff
@@ -0,0 +1,57 @@
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} Physical cluster replication uses two clusters with _cluster virtualization_ enabled to stream data from an active primary cluster to a passive standby cluster. Each cluster contains: a _system interface_ and an application _virtual cluster_: {% comment %}Link to Matt's work here on general cluster virtualization reference{% endcomment %}
+
+{% include {{ page.version.version }}/physical-replication/interface-virtual-cluster.md %}
+
+This separation of concerns means that the replication stream can operate without affecting work happening in the virtual cluster.
+
+### Start the replication stream
+
+When you initiate physical cluster replication, it will replicate the application virtual cluster from the primary to the standby cluster. The replication stream sets up a series of jobs that complete the following work:
+
+1. The standby's system interface communicates to the primary cluster that a stream should start.
+1. The primary cluster chooses a timestamp at which to start the stream and lays down a [protected timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#protected-timestamps), which will protect the entirety of the cluster's data from [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) until an initial scan is complete.
+1. The initial scan runs on the primary and backfills all data from the primary virtual cluster up to the starting timestamp of the replication stream.
+1. The primary cluster returns the timestamp and a [job ID]({% link {{ page.version.version }}/show-jobs.md %}#response) for the replication job.
+1. The standby cluster uses a PG connection to retrieve all of the primary cluster's node addresses. This generates a topology of the primary cluster. If there is a mismatch in nodes between the primary and standby, the replication job will share out the work between the standby's nodes.
+1. The stream from the primary virtual cluster begins with the standby's system interface ingesting the data into its own virtual cluster.
+
+<img src="{{ 'images/v23.2/physical-rep-to.png' | relative_url }}" alt="Two virtualized clusters with system and application tenants showing the directional stream." style="border:0px solid #eee;max-width:100%" />
+
+### During the replication stream
+
+The replication happens at the byte level, which means that the job is unaware of databases, tables, row boundaries, and so on. However, when a [cutover](#cutover-and-promotion-process) to the standby cluster is initiated, the replication job ensures that the cluster is in a transactionally consistent state as of a certain point in time.
+
+{% comment %} Add:  no need to coordinate with any schema changes. Don't know anything about zone configurations will be stored using the default zone config.-->{% endcomment %}
```

Comment:
As of 23.1 we will have basic zone configuration replication as part of the stream.

Reply:
I have added a sentence here along with some of the other details I hadn't included re what is also replicated. I may have missed something important from this list though?